docs/src/icons/SvgHamburgerMenu.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+function SvgComponent(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg
```

Comment:
How about we animate it into an X when opened? :)

Reply:
Yeah, I thought about that too.